src/symmetry/symmetrize_occupation_matrix.hpp

```diff
@@ -46,6 +48,19 @@ symmetrize_occupation_matrix(Occupation_matrix& om__)
         }
     }
 
+    std::vector<std::vector<mdarray<double, 2>>> rotms(sym.size());
```

Comment:
but why not passing it as argument? it is already computed in sim.ctx.

Reply:
Aren't the ones in sim ctx different? https://github.com/electronic-structure/SIRIUS/blob/sirius-md/src/context/simulation_context.cpp#L834

It does the following at the end:
https://github.com/electronic-structure/SIRIUS/blob/sirius-md/src/core/sht/sht.cpp#L98

In `symmetrize_occupation_matrix` for each symmetry 
https://github.com/electronic-structure/SIRIUS/blob/sirius-md/src/core/sht/sht.cpp#L115
was called, which returns a vector of rotations matrices.

I didn't look at the for loops below, perhaps one can rearrange things and use the ctx.rotm().

Reply:
I checked the code and I found a TODO that perfrcty fits this refactoring. I'll add my changes to this PR